backend/census_historical_migration/migration_result.py

```diff
@@ -0,0 +1,5 @@
+"""Stores the result for the current migration"""
+
+DEFAULT_RESULT = {"success": [], "errors": [], "transformations": []}
+
+result = DEFAULT_RESULT
```

Comment:
How about something like 
```
class MigrationResult:
    DEFAULT_RESULT = {"success": [], "errors": [], "transformations": []}
    result = DEFAULT_RESULT.copy()

    @staticmethod
    def reset():
        MigrationResult.result = MigrationResult.DEFAULT_RESULT.copy()

    @staticmethod
    def update_transformation(data):
        MigrationResult.result["transformations"].append(data)
    ....................an update function for errors and success .... or something similar

Reply:
Done (I called them `append_*` to make sure it's clear)